java/src/main/java/ai/rapids/cudf/ColumnWriterOptions.java

```diff
@@ -143,6 +144,22 @@ protected ColumnWriterOptions withTimestamp(String name, boolean isInt96,
       return new ColumnWriterOptions(name, isInt96, UNKNOWN_PRECISION, isNullable, parquetFieldId);
     }
 
+    protected ColumnWriterOptions withBinary(String name, boolean isNullable) {
+      ColumnWriterOptions opt = listBuilder(name, isNullable)
+          .withColumns(false, "BINARY_DATA")
```

Comment:
Nit: It would be nice to have a comment here explaining why we're hardcoding a name here and whether the name used here is significant in any way.  Also applies to the overload below.